terraform/projects/app-ecs-services/prometheus-service.tf

```diff
@@ -3,6 +3,63 @@
 *
```

Comment:
Could we expand this description to include the added/changed functionality please